third_party/Makefile

```diff
@@ -39,6 +39,9 @@ build-extlib:
 build-record-update:
 	$(MAKE) -C coq-record-update
 
+build-koika:
+	$(MAKE) -C koika
```

Comment:
Does this change add a Koika dependency?

Reply:
Ahh this leaked in as I was building koika but dont intend to add it to third_party anymore: For Koika (and possibly other opam'able modules like coq-ext-lib/coq-record-update, when they have up to date releases and aren't making modifications) my intention is to require they be opam installed rather a custom build, only vendoring if we making changes.

Reply:
I think that would be a great change!